src/frontend/src/App.jsx

```diff
@@ -6,13 +6,14 @@ import { BrowserRouter as Router } from 'react-router-dom';
 import Header from './header/index';
 import Routers from './routers';
 import CookieBanner from './cookieBanner/cookie';
-import './testServer';
+// import './testServer';
 import './App.scss';
 import './locales/i18n';
 
 function App() {
   const [authorization, onAuthorization] = useState(false);
   const [popupLogin, changePopup] = useState(false);
+  const [language, changeLanguage] = useState({ value: 'UA', label: 'UA' });
```

Comment:
Please, use code style as i mentioned previously - 
in stead of const [language, changeLanguage] = useState({ value: 'UA', label: 'UA' });
use 
const [language, setLanguage] = useState({ value: 'UA', label: 'UA' }); and in many other places make changes as i wrote here